src/main/java/edu/kit/hci/soli/controller/UsersController.java

```diff
@@ -0,0 +1,62 @@
+package edu.kit.hci.soli.controller;
+
+import edu.kit.hci.soli.config.security.SoliUserDetails;
+import edu.kit.hci.soli.domain.User;
+import edu.kit.hci.soli.dto.KnownError;
+import edu.kit.hci.soli.service.BookingsService;
+import edu.kit.hci.soli.service.UserService;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+
+@Controller
+@Slf4j
+public class UsersController {
+
+    private final UserService userService;
+
+    public UsersController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PutMapping("/admin/users/deactivate/{userId}")
```

Comment:
IMO this should be `users/{userId}/deactivate` to more cleanly align with the path design we use elsewhere

Reply:
Fair enough